eth-custodian/contracts/ProofKeeper.sol

```diff
@@ -29,28 +42,56 @@ contract ProofKeeper {
 
     /// Parses the provided proof and consumes it if it's not already used.
     /// The consumed event cannot be reused for future calls.
-    function _parseAndConsumeProof(bytes memory proofData, uint64 proofBlockHeight)
+    function _parseAndConsumeProof(
+        bytes memory proofData, 
+        uint64 proofBlockHeight
+    )
         internal
-        returns (ProofDecoder.ExecutionStatus memory result)
+        returns(ProofDecoder.ExecutionStatus memory result)
     {
-        require(proofBlockHeight >= minBlockAcceptanceHeight_, "Proof is from the ancient block");
-        require(prover_.proveOutcome(proofData, proofBlockHeight), "Proof should be valid");
+        require(
+            proofBlockHeight >= minBlockAcceptanceHeight_,
+            'Proof is from the ancient block'
+        );
+        require(
+            prover_.proveOutcome(proofData,proofBlockHeight),
+            'Proof should be valid'
+        );
 
         // Unpack the proof and extract the execution outcome.
         Borsh.Data memory borshData = Borsh.from(proofData);
-        ProofDecoder.FullOutcomeProof memory fullOutcomeProof = borshData.decodeFullOutcomeProof();
-        require(borshData.finished(), "Argument should be exact borsh serialization");
 
-        bytes32 receiptId = fullOutcomeProof.outcome_proof.outcome_with_id.outcome.receipt_ids[0];
-        require(!usedEvents_[receiptId], "The burn event cannot be reused");
+        ProofDecoder.FullOutcomeProof memory fullOutcomeProof = 
+        borshData.decodeFullOutcomeProof();
+        
+        require(
+            borshData.finished(),
+            'Argument should be exact borsh serialization'
+        );
+
+        bytes32 receiptId = 
+        fullOutcomeProof.outcome_proof.outcome_with_id.outcome.receipt_ids[0];
+
+        require(
+            !usedEvents_[receiptId],
+            'The burn event cannot be reused'
+        );
         usedEvents_[receiptId] = true;
 
-        require(keccak256(fullOutcomeProof.outcome_proof.outcome_with_id.outcome.executor_id)
-                == keccak256(nearProofProducerAccount_),
-                "Can only withdraw coins from the linked proof producer on Near blockchain");
+        require(
+            keccak256(fullOutcomeProof.outcome_proof.outcome_with_id.outcome.executor_id) == 
+            keccak256(nearProofProducerAccount_),
```

Comment:
`           keccak256(fullOutcomeProof.outcome_proof.outcome_with_id.outcome.executor_id)
            == keccak256(nearProofProducerAccount_),`

If we leave the equal sign it on the previous line that has a negative effect on the readability and will look like this is just another argument to the function.

Reply:
I agree ... I only did that because the linter had a warning by adding the == in the next line.

A good way is to avoid this by assigning the two parties to a memory parameters.
```javascript
param1 = keccak256(....x1);
param2 = keccak256(....x2);
require(param1 == param2, error_msg);
```